mission-5/src/dto/users.dto.ts

```diff
@@ -0,0 +1,19 @@
+import type { GetListParams, Options, singularContentType, UserId } from '@/types/shared.type.js';
```

Comment:
singularContentType 타이포 에러로 빌드가 실패합니다!

```suggestion
import type { GetListParams, Options, SingularContentType, UserId } from '@/types/shared.type.js';
```